### PR TITLE
Implement cSpell to detect spelling mistakes in codebase

### PR DESCRIPTION
**Pull request by laurent22**

### CODE REVIEW

File: readme/dev/coding_style.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ const config: Config = {
 
 ### Don't set the type when it can be inferred
 
-TypeScript can automatically detect the type so setting it explicitely in many cases is not needed, and makes the code unecessary verbose. We already have enabled the eslint rule `no-inferrable-types`, however it only applies to simple types such as string, number, etc. but not to function calls.
+TypeScript can automatically detect the type so setting it explicitly in many cases is not needed, and makes the code unnecessary verbose. We already have enabled the eslint rule `no-inferrable-types`, however it only applies to simple types such as string, number, etc. but not to function calls.
```
Comment (personalizedrefrigerator) [inline readme/dev/coding_style.md]:
```suggestion
TypeScript can automatically detect the type so setting it explicitly in many cases is not needed, and makes the code unnecessarily verbose. We already have enabled the eslint rule `no-inferrable-types`, however it only applies to simple types such as string, number, etc. but not to function calls.
```

File: packages/utils/url.ts [hunk 2]
```diff
@@ -8,7 +8,7 @@
 // - It always returns paths with forward slashes "/". This is normally handled
 //   properly everywhere.
 //
-// - Adds the "platform" parameter to optionall return paths with "\" for win32
+// - Adds the "platform" parameter to optional return paths with "\" for win32
```
Comment (personalizedrefrigerator) [inline packages/utils/url.ts]:
```suggestion
// - Adds the "platform" parameter to optionally return paths with "\" for win32
```

File: packages/tools/locales/en_US.po [hunk 3]
```diff
@@ -2954,7 +2954,7 @@ msgid "Nord"
 msgstr ""
 
 #: packages/app-cli/app/command-sync.ts:89
-msgid "Not authentified with %s. Please provide any missing credentials."
+msgid "Not authenticated with %s. Please provide any missing credentials."
 msgstr "Not authenticated with %s. Please provide any missing credentials."
```
Comment (tessus) [inline packages/tools/locales/en_US.po:2958]: Haha, I guess I can remove my translation again. ;-)

**Comment by laurent22** [comment]
CI is passing so looks like all is good. Thanks for the extra fixes!

**Pull request merged**